backend/service/aws/dynamodb.go

```diff
@@ -118,11 +118,33 @@ func newProtoForIndexStatus(s types.IndexStatus) dynamodbv1.GlobalSecondaryIndex
 	return dynamodbv1.GlobalSecondaryIndex_Status(value)
 }
 
-func newProtoForBillingMode(s types.BillingMode) dynamodbv1.Table_BillingMode {
-	value, ok := dynamodbv1.Table_BillingMode_value[string(s)]
+// manually check the billing mode by inferring it from throughput
+// to cover cases where AWS does not return the mode in the table description
+// if a table is PROVISIONED, it will have at least 1 RCU/WCU provisioned
+// if a table is PAY_PER_REQUEST (on demand), it will have 0 RCU/WCU provisioned
+func checkBillingMode(t *types.ProvisionedThroughputDescription) types.BillingMode {
+	if (*t.ReadCapacityUnits > 0) || (*t.WriteCapacityUnits > 0) {
+		return "PROVISIONED"
```

Comment:
Shouldn't we have constants for this already defined in proto? Can we the string value there instead of writing them out here? This applies for all use cases in this pr.